src/blender/creating-model-in-blender.md

```diff
@@ -2,4 +2,18 @@
 
 ## Beginning
```

Comment:
I feel better removing this file as a whole - it will be then obvious that the article is not ready

Reply:
You can just remove the link to this page for now. I left it blank just because I'll fill it out later.